mlxtend/frequent_patterns/tests/test_fpbase.py

```diff
@@ -149,6 +157,29 @@ def test_with_fill_values(fill_value):
         test_with_fill_values(0)
         test_with_fill_values(False)
 
+    def test_sparse(self):
+        def test_with_fill_values(fill_value):
+            try:
+                sdt = pd.SparseDtype(type(fill_value), fill_value=fill_value)
+                sdf = self.df.astype(sdt)
+            except AttributeError:
+                # pandas < 0.25
```

Comment:
see the ` from distutils.version import LooseVersion as Version` approach mentioned above